APIScrape/scrape.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+require_once "../Utilities/DB.php";
+
+define('API_KEY', 'x-api-key: ec254e44-3996-458b-8522-4933954d8fcd');
+
+$db = new DB();
+$dbConnection = $db->dbConnectToHostOnly();
+
+/**
+ * @param PDO $db Takes DB containing a connection to the database.
+ *  * Creates a new database into which to put the scraped data.
+ * If the database already exists, tears it down and creates a new one.
+ */
+function createDatabase(PDO $db) {
+    try {
+        $sql = "DROP DATABASE IF EXISTS `cat-test`;
```

Comment:
This name was not signed off in db architecture, and it's not an appropriate name for the database because it's not a test. Please change it - suggest `cats`.

Reply:
Cmd + Shift + R - `cat-test` -> `cat`

Reply:
on it

Reply:
On it

Reply:
done